### ISSUE TITLE
Defining custom features

### ISSUE THREAD

**Issue by cifkao** [jeffreyjohnens/style_rank#1]
Is it possible to define custom features in Python, without recompiling the StyleRank C++ code? For example, do you have code to use jSymbolic features as described in the paper?

**Comment by cifkao** [comment]
OK, I guess the way to pass custom features is via `raw_features`. :slightly_smiling_face:

**Comment by jeffreyjohnens** [comment]
Yeah passing in raw_features would probably be the best way to do it if you don't want to write custom features in c++. Let me know if you run into any issues.

Thanks for catching the issue with the missing import!
